buildozer/default.spec

```diff
@@ -162,6 +162,11 @@ fullscreen = 0
 # (list) Gradle dependencies to add
 #android.gradle_dependencies =
 
+# (bool) Enable AndroidX support. Enable when 'android.gradle_dependencies'
+# contains an 'androidx' package, or any package from Kotlin source.
+# android.enable_androidx requires android.api >= 28 
```

Comment:
maybe pep8 is unhappy about the trailing space
```suggestion
# android.enable_androidx requires android.api >= 28
```

Reply:
Thank you. It was trailing whitespace.